integration/test_ft_search_infields.py

```
@@ -0,0 +1,356 @@
+            "title", "TEXT",
+        )
+        client.execute_command("HSET", "err:1", "title", "apple banana")
+        IndexingTestHelper.wait_for_backfill_complete_on_node(client, "err_idx")
+
+        try:
+            client.execute_command(
+                "FT.SEARCH", "err_idx", "apple",
+                "INFIELDS", "abc",
+                "DIALECT", "2",
+            )
+            assert False, "Expected error for non-integer INFIELDS count"
```

Ruff (0.15.20)

[warning] Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)